5 дз/5 дз/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿//Methods
+
+using System;
+
+namespace Methods
+{
+    class Program
+    {
+        static double Count(int a, int b)
+        {
+            return ((Math.Sqrt(a) + a) / b);
+        }
+
+        public static void Main()
+        {
+            Console.WriteLine("Значение выражения равно = " + (Count(6, 2) + Count(13, 5) + Count(21, 3)));
```

Comment:
В задании требуется

> результат выведите на консоль с точностью до трех знаков после запятой

То есть так (округление)
```suggestion
            Console.WriteLine("Значение выражения равно = " + Math.Round((Count(6, 2) + Count(13, 5) + Count(21, 3)), 3));
```
А еще лучше так (форматирование). Читать и понимать проще. Да и две задачи (вычисление значения выражения и вывод результата на печать) разделены на два оператора.
```suggestion
            double y = Count(6, 2) + Count(13, 5) + Count(21, 3);
            Console.WriteLine($"Значение выражения равно = {y:F3}");
```